pom.xml

```diff
@@ -2405,11 +2407,39 @@
             </execution>
           </executions>
         </plugin>
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>build-helper-maven-plugin</artifactId>
-          <version>3.0.0</version>
-        </plugin>
+	<plugin>
+	  <groupId>org.codehaus.mojo</groupId>
```

Comment:
@holdenk, just to understand more about the PR, what does this PR do for the actual builds? Does it set a new property `module.build.timestamp` available in the maven build, and you plan to leverage this?

Reply:
It allows the user to supply a timestamp, but more importantly it ensures the sub-modules are built with the same timestamp assigned for build versioning rather than each sub modules build start time.

Reply:
Yeah, I double checked why `module.build.timestamp` was added at https://github.com/mojohaus/build-helper-maven-plugin/pull/32. The purpose seems good but I was more trying to understand what issue this PR fixes in the current build.

Is the timestamp used somewhere in the Spark build?

Reply:
If someone uses `mvn deploy` it is used in generating the snapshot version tag.

Reply:
Ohh got it. Sounds good. Shall we describe the diff before/after results in PR description?